### ISSUE TITLE
Simultaneous keystrokes (with non-modifier keys)

### ISSUE THREAD

**Issue by NMichas** [BlueM/cliclick#109]
Hello, is it possible to send two (or more) simultaneous keystrokes of just any individual keys?

For example, something like` t:eu` but instead of 'typing' the two letters one after the other to actually press the keys simultaneously?

Thanks!

p.s. I'm using clickclick with ShockEmu to emulate sending commands to my PS4 controller, where I need to activate a combo-command requiring to press two keys of the virtual gamepad simultaneously.

**Comment by BlueM** [comment]
No, currently not – this is not a very frequent use case, so no one has shown interest in this, at least until now.

Technically, this should be possible to do. I’d guess the most obvious way to implement this would be to extend `KeyDownAction`’s functionality to non-modifier keys. If you’re willing to submit a PR, I’ll take a look.
